src/tests/system/tests/test_passkey.py

```diff
@@ -425,3 +425,83 @@ def test_passkey__su_same_key_for_multi_user(
             ioctl=f"{moduledatadir}/umockdev.ioctl",
             script=f"{testdatadir}/umockdev.script.{suffix}.{user}",
         )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.ticket(jira="SSSD-7011", gh=7066)
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.builtwith(client="passkey", provider="passkey")
+def test_passkey__ssh_key_as_passkey_mapping_token(
+    client: Client, provider: GenericADProvider, moduledatadir: str, testdatadir: str
```

Comment:
I think you need to replace `provider: GenericADProvider` by `provider: GenericProvider` to make CI happy